report/renderer/util.js

```diff
@@ -417,7 +417,7 @@ export class Util {
             throttling.downloadThroughputKbps === 1.6 * 1024 * 0.9 &&
             throttling.uploadThroughputKbps === 750 * 0.9;
         };
-        summary = `${isSlow4G() ? 'Slow 4G' : 'Custom'} throttling by DevTools`;
+        summary = isSlow4G() ? Util.i18n.strings.runtimeSlow4g : Util.i18n.strings.runtimeCustom;
```

Comment:
Ah perfect, now I can use this util function directly in https://github.com/GoogleChrome/lighthouse/pull/13305!